dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java

```diff
@@ -0,0 +1,160 @@
+package com.dotcms.contenttype.business;
+
+import com.dotcms.contenttype.business.FieldAPIImpl;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.contenttype.transform.contenttype.ContentTypeInternationalization;
+import com.dotcms.languagevariable.business.LanguageVariableAPI;
+import com.dotcms.system.event.local.business.LocalSystemEventsAPI;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.PermissionAPI;
+import com.dotmarketing.business.RelationshipAPI;
+import com.dotmarketing.business.UserAPI;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.portlets.contentlet.business.ContentletAPI;
+import com.liferay.portal.model.User;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.net.ConnectException;
+import java.util.Map;
+
+import static com.dotcms.util.CollectionsUtils.map;
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.junit.Assert.assertEquals;
+
+public class FieldAPIImplTest {
+    private static PermissionAPI perAPI;
+    private static ContentletAPI conAPI;
+    private static UserAPI userAPI;
+    private static RelationshipAPI relationshipAPI;
+    private static LocalSystemEventsAPI localSystemEventsAPI;
+    private static LanguageVariableAPI languageVariableAPI;
+
+    private static FieldAPIImpl fieldAPIImpl;
+
+    @BeforeClass
+    public static void init(){
+        perAPI = mock(PermissionAPI.class);
+        conAPI = mock(ContentletAPI.class);
+        userAPI = mock(UserAPI.class);
+        relationshipAPI = mock(RelationshipAPI.class);
+        localSystemEventsAPI = mock(LocalSystemEventsAPI.class);
+        languageVariableAPI = mock(LanguageVariableAPI.class);
+
+        fieldAPIImpl = new FieldAPIImpl(
+                perAPI,
+                conAPI,
+                userAPI,
+                relationshipAPI,
+                localSystemEventsAPI,
+                languageVariableAPI
+        );
+    }
+
+    /**
+     * when: you have to language variable for the Content Type's fields
+     * Should: Change the field property name by the language variable values
+     */
+    @Test
+    public void shouldFieldInternationalizationWork(){
+        final long languageId = 1;
+        final boolean live = true;
+        final User user = mock(User.class);
+        final Object object = mock(Object.class);
+
+        final ContentType contentType = mock(ContentType.class);
+        final ContentTypeInternationalization contentTypeInternationalization = mock(ContentTypeInternationalization.class);
+        final Map<String, Object> fieldMap = map(
+                "attribute_1", "value_1",
+                "attribute_2", "value_2",
+                "attribute_3", "value_3",
+                "variable", "testField",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "variable" appears 4 times in this file; the first occurrence is on line 73](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)